### DISCUSSION TITLE
Rules for contribution

### DISCUSSION THREAD

**Discussion by xmba15** [STVIR/pysot#473] [General]
Hi, I have tried the demo script and to me, this package has high potentials not only for research but also for industry use too.
If able, I would like to contribute to make the codes cleaner and maybe easier for setup, usage.
Can any maintainers write a rule for contributors in terms of styles of code, how to contribute, introduction of python formatter like pylint or black...
Would appreciate if current maintainers can reply for this.

**Comment by ZhiyuanChen** [comment]
> Hi, I have tried the demo script and to me, this package has high potentials not only for research but also for industry use too.
> If able, I would like to contribute to make the codes cleaner and maybe easier for setup, usage.
> Can any maintainers write a rule for contributors in terms of styles of code, how to contribute, introduction of python formatter like pylint or black...
> Would appreciate if current maintainers can reply for this.

Thank you very much for your intention. 

As the deadline of CVPR is approaching, it would be very unlikely for us to provide a `code of conduct` right away, but we will ensure to provide it right after the deadline. 

I would also suggest you to check out the styling guide of [MMCV](https://github.com/open-mmlab/MMCV), as we should have very similar requirements if you are in hurry.

**Reply by xmba15** [reply]
@ZhiyuanChen thanks for the reply. I will check the styling guide of MMCV and see through the code base of pysot to see if I can help improve any parts.
